keyboardManager/src/commonMain/kotlin/KeyboardManager.kt

```diff
@@ -0,0 +1,66 @@
+package com.splendo.kaluga.keyboardManager
+
+/*
+
+Copyright 2019 Splendo Consulting B.V. The Netherlands
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+expect class KeyboardView {
```

Comment:
Maybe try `expect typealias`?

Reply:
I have association that `KeyboardView` is actual view of the keyboard itself

Reply:
Yes we had the same idea. @Daeda88 will try it.

Reply:
Eventually we should have this in `base` or maybe a `ui` module.